compose/service.py

```diff
@@ -57,6 +57,7 @@
     'volumes_from',
     'security_opt',
     'cpu_quota',
+    'shm_size',
```

Comment:
This list is supposed to be in sorted order.

I see now that the last couple additions were not adding in order. Would you mind running a `:sort` on this section to move the `shm_size` into the right place (and the other two at the same time).